butte-build/src/ir/keywords.rs

```diff
@@ -0,0 +1,56 @@
+pub fn substitute_keyword(input: &str) -> Option<&'static str> {
+    Some(match input {
+        "as" => "as_",
```

Comment:
sort this

Reply:
That list is sorted, but there are two sublists, current keywords and reserved keywords, though I just noticed the list is outdated and does not include edition 2018.
Fixed by updating the list and adding a comment.